### PR TITLE
gha configurations

### PR DESCRIPTION
**Pull request by dalerichardson**

### CODE REVIEW

File: _states/win_lgpo_ash.py [hunk 1]
```diff
@@ -76,70 +82,62 @@ def present(name, mode=None, value=None, vtype=None, policies=None, **kwargs):
...
                 key: HKLM\Software\Salt\Foo
                 value: 0
                 vtype: REG_DWORD
-              - policy_type: regpol
+                - policy_type: regpol
                 key: HKLM\Software\Salt\Bar
                 value: 0
                 vtype: REG_DWORD
-              - policy_type: secedit
+                - policy_type: secedit
```
Comment (lorengordon) [inline _states/win_lgpo_ash.py:108]: Aw shucks. Unfortunately, this is not actually python, this is a docstring. It is yaml embedded in rst. Which means these new indents, from line 85-108, are not valid. I'm not super sure how to handle that with editorconfig, so for now, revert these indents, and add a rules that unsets the failing rules for this file.

File: ash-windows/mss/Templates/MSS-legacy.admx [hunk 2]
```diff
@@ -276,7 +276,7 @@ MACHINE\System\CurrentControlSet\Services\Tcpip\Parameters\EnableDeadGWDetect,4,
 MACHINE\System\CurrentControlSet\Services\Tcpip\Parameters\EnableICMPRedirect,4,%EnableICMPRedirect%,0
 MACHINE\System\CurrentControlSet\Services\Lanmanserver\Parameters\Hidden,4,%HideFromBrowseList%,0
 MACHINE\System\CurrentControlSet\Services\Tcpip\Parameters\KeepAliveTime,4,%KeepAliveTime%,3,150000|%KeepAliveTime0%,300000|%KeepAliveTime1%,600000|%KeepAliveTime2%,1200000|%KeepAliveTime3%,2400000|%KeepAliveTime4%,3600000|%KeepAliveTime5%,7200000|%KeepAliveTime6%
-MACHINE\System\CurrentControlSet\Services\IPSEC\NoDefaultExempt,4,%NoDefaultExempt%,3,0|%NoDefaultExempt0%,1|%NoDefaultExempt1%,2|%NoDefaultExempt2%,3|%NoDefaultExempt3% 
+MACHINE\System\CurrentControlSet\Services\IPSEC\NoDefaultExempt,4,%NoDefaultExempt%,3,0|%NoDefaultExempt0%,1|%NoDefaultExempt1%,2|%NoDefaultExempt2%,3|%NoDefaultExempt3
```
Comment (lorengordon) [inline ash-windows/mss/Templates/MSS-legacy.admx:279]: This file should not have been modified
```suggestion
MACHINE\System\CurrentControlSet\Services\IPSEC\NoDefaultExempt,4,%NoDefaultExempt%,3,0|%NoDefaultExempt0%,1|%NoDefaultExempt1%,2|%NoDefaultExempt2%,3|%NoDefaultExempt3% 
```

File: ash-windows/mss/Templates/MSS-legacy.admx [hunk 3]
```diff
@@ -287,6 +287,6 @@ MACHINE\System\CurrentControlSet\Services\Tcpip\Parameters\TcpMaxConnectResponse
 MACHINE\System\CurrentControlSet\Services\Tcpip\Parameters\TcpMaxDataRetransmissions,4,%TcpMaxDataRetransmissions%,1
 MACHINE\SYSTEM\CurrentControlSet\Services\Eventlog\Security\WarningLevel,4,%WarningLevel%,3,50|%WarningLevel0%,60|%WarningLevel1%,70|%WarningLevel2%,80|%WarningLevel3%,90|%WarningLevel4%
 MACHINE\System\CurrentControlSet\Services\Tcpip6\Parameters\DisableIPSourceRouting,4,%DisableIPSourceRoutingIPv6%,3,0|%DisableIPSourceRouting0%,1|%DisableIPSourceRouting1%,2|%DisableIPSourceRouting2% 
-MACHINE\System\CurrentControlSet\Services\Tcpip6\Parameters\TcpMaxDataRetransmissions ,4,%TcpMaxDataRetransmissionsIPv6%,1
+MACHINE\System\CurrentControlSet\Services\Tcpip6\Parameters\TcpMaxDataRetransmissions ,4,%TcpMaxDataRetransmissionsIPv6%,
```
Comment (lorengordon) [inline ash-windows/mss/Templates/MSS-legacy.admx:290]: File should not be modified

```suggestion
MACHINE\System\CurrentControlSet\Services\Tcpip6\Parameters\TcpMaxDataRetransmissions ,4,%TcpMaxDataRetransmissionsIPv6%,1
```

File: ash-windows/mss/Templates/MSS-legacy.admx (hunk 2 above)
Comment (dalerichardson) [reply ash-windows/mss/Templates/MSS-legacy.admx]: Done

File: _states/win_lgpo_ash.py (hunk 1 above)
Comment (dalerichardson) [reply _states/win_lgpo_ash.py]: Changes made

File: ash-windows/mss/Templates/MSS-legacy.admx [hunk 4]
```diff
@@ -276,7 +276,7 @@ MACHINE\System\CurrentControlSet\Services\Tcpip\Parameters\EnableDeadGWDetect,4,
 MACHINE\System\CurrentControlSet\Services\Tcpip\Parameters\EnableICMPRedirect,4,%EnableICMPRedirect%,0
 MACHINE\System\CurrentControlSet\Services\Lanmanserver\Parameters\Hidden,4,%HideFromBrowseList%,0
 MACHINE\System\CurrentControlSet\Services\Tcpip\Parameters\KeepAliveTime,4,%KeepAliveTime%,3,150000|%KeepAliveTime0%,300000|%KeepAliveTime1%,600000|%KeepAliveTime2%,1200000|%KeepAliveTime3%,2400000|%KeepAliveTime4%,3600000|%KeepAliveTime5%,7200000|%KeepAliveTime6%
-MACHINE\System\CurrentControlSet\Services\IPSEC\NoDefaultExempt,4,%NoDefaultExempt%,3,0|%NoDefaultExempt0%,1|%NoDefaultExempt1%,2|%NoDefaultExempt2%,3|%NoDefaultExempt3% 
+MACHINE\System\CurrentControlSet\Services\IPSEC\NoDefaultExempt,4,%NoDefaultExempt%,3,0|%NoDefaultExempt0%,1|%NoDefaultExempt1%,2|%NoDefaultExempt2%,3|%NoDefaultExempt3%
```
Comment (lorengordon) [inline ash-windows/mss/Templates/MSS-legacy.admx:279]: This file should not show any changes. Looks like a space was removed at the end of the line. The space should be restored.

Comment (dalerichardson) [reply ash-windows/mss/Templates/MSS-legacy.admx]: Space has been restored

**Review by lorengordon — APPROVED** [review]

**Pull request merged**